app/resources/views/pages/location_search/spreadsheet.blade.php

```diff
@@ -15,7 +15,8 @@ class="form-control" id="distance"
 			<button type="button" id="updateRadius" class="btn btn-primary">Update</button>
 	</div>
 	@if (count($locations) === 0)
-		No location found matching the specified keywords
+		<h3>No location found matching the specified keywords</h3>
+		<a class="btn btn-default" href="/locations-added-by-me">Add New Location</a>
```

Comment:
Navigating directly to /add-location would be quicker for people who want to add locations.

Also, don't show this only when the user is signed in.  If they're not signed in, they won't be able to add a location which would make the link confusing.

Did you test by shrinking the width of the browser?  I didn't test this yet but that is a quick way to simulate what various mobile devices will see.  If you see anything wrong, you can update the corresponding .scss file in resources/assets/sass directory.